.hermes/conveyor/work-7c16e7ae/initial_plan.md

```
@@ -0,0 +1,114 @@
+- `givenwhen` — `given ($1) { when ($2) { $0 } default { } }`
+
+**Try/Catch (5.34+):**
+- `trycatch` — `try { $1 } catch ($e) { $0 }`
+- `tryfinally` — `try { $1 } finally { $0 }`
+
+**Moo/Moose Method Modifiers:**
+- `haslazy` — `has '${1:attr}' => ( is => 'rw', lazy => 1, builder => '_build_${1:attr}' );`
+- `hasbuilder` — `_build_${1:attr} method`
+- `aroundmod` — `around ${1:method} { my ($self, @args) = @_; $0 }`
+- `beforemod` — `before ${1:method} => sub { $0 };`
+- `aftermof` — `after ${1:method} => sub { $0 };`
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)